raven/src/main/java/org/opencadc/raven/RavenInitAction.java

```diff
@@ -150,6 +148,8 @@ void initDAO() {
         Map<String,Object> dc = getDaoConfig(props);
         ArtifactDAO artifactDAO = new ArtifactDAO();
         artifactDAO.setConfig(dc); // connectivity tested
+        PreauthKeyPairDAO kpDAO = new PreauthKeyPairDAO();
```

Comment:
this is not necessary since instantiating the artifactDAO is enough to check the db connection; that's all that's happening here